content/rooms/fragments/v11-redactions.md

```diff
@@ -0,0 +1,38 @@
+---
+---
+
+{{% added-in this=true %}} The top-level `origin`, `membership`, and `prev_state` properties
```

Comment:
The `added-in` template doesn't seem to be rendering properly here?

Reply:
https://github.com/matrix-org/matrix-spec/pull/1604#discussion_r1287317592